kymatio/scattering1d/filter_bank.py

```diff
@@ -253,7 +261,7 @@ def compute_sigma_psi(xi, Q, r=math.sqrt(0.5)):
     return xi * term1 * term2
 
 
-def compute_temporal_support(h_f, criterion_amplitude=1e-3):
+def compute_temporal_support(h_f, criterion_amplitude=1e-3, warn=False):
```

Comment:
dare I say that `criterion_amplitude` is unlikely to change? hence, when will `warn` be necessary? for who is its exposure useful?

Reply:
The behavior of `warn=True` may be obtained via a Python context manager:

```
with warnings.catch_warnings():
    warning.simplefilter('ignore')
```

so i don't think this `warn=False` new kwarg is necessary.

I don't fully understand the implications of `criterion_amplitude`.